include/ignition/physics/Joint.hh

```diff
@@ -26,6 +26,10 @@ namespace ignition
   namespace physics
   {
     /////////////////////////////////////////////////
+    /// \brief This feature retrieves the generalized Joint states such as
+    /// Position, Velocity, Acceleration of the Joint, applied Force to the
+    /// Joint and the transformation matrix from the Joint's parent Link to this
+    /// Joint's child Link based on its current position.
```

Comment:
We can just use lower case to refer to the class names to keep documentation consistent.